gadget/validate.go

```diff
@@ -95,6 +96,13 @@ func ruleValidateVolumes(vols map[string]*Volume, model Model, extra *Validation
 		// if system-seed role is mentioned assume the uc20
```

Comment:
do we need a new validation check that system-seed an system-seed-null are not used at the same time across volumes?

Reply:
Good point, I've added that test